src/libs/actions/User.ts

```diff
@@ -469,6 +471,56 @@ function triggerNotifications(onyxUpdates: OnyxServerUpdate[]) {
     });
 }
 
+function playSoundForMessageType(pushJSON: OnyxServerUpdate[]) {
+    try {
+        const reportActionsOnly = pushJSON.filter((update) => update.key.includes('reportActions_'));
+        const flatten = flattenDeep(reportActionsOnly.map((update) => Object.keys(update.value).map((key) => update.value[key])));
+        const types = flatten.map((data) => data.originalMessage).filter(Boolean);
+
+        // Someone completes a task
+        if (flatten.find((data) => data.actionName === 'TASKCOMPLETED')) {
+            return playSound('success');
+        }
+
+        // Someone completes a money request
+        if (flatten.find((data) => data.actionName === 'IOU')) {
+            return playSound('success');
+        }
+
+        // someone sent money
+        if (types.find((message) => message.IOUDetails)) {
+            return playSound('success');
+        }
+
+        // mention user
+        if (types.find((message) => message.html?.includes('<mention-user>'))) {
+            return playSound('attention');
+        }
+
+        // mention @here
+        if (types.find((message) => message.html?.includes('<mention-here>'))) {
+            return playSound('attention');
+        }
+
+        // assign a task
+        if (types.find((message) => message.taskReportID)) {
+            return playSound('attention');
+        }
+
+        // request money
+        if (types.find((message) => message.IOUTransactionID)) {
+            return playSound('attention');
+        }
+
+        // plain message
+        if (types.find((message) => message.html)) {
+            return playSound('receive');
+        }
+    } catch (e) {
+        // do nothing
```

Comment:
Let's log the error at least with `import Log from '@libs/Log';` using Log.client